notebooks/book2/02/ecdf_sample.ipynb

```diff
@@ -1,97 +1,220 @@
 {
```

Comment:
<p>This should not be the title. Instead just include origijnal comment</p><p><code></code>`</p><pre class="ql-syntax" spellcheck="false"># empirical cdf
# Source: fig 11.17 of 'Bayesian Modeling and Computation<code></code>`
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probml/pyprobml/pull/989/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Done changes.